source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1258,17 +1258,15 @@ def isochrone_common(isochrone, request, instance, journey_req):
     journey_req.streetnetwork_params.destination_mode = isochrone.destination_modes[0]
 
 
-def is_olympic_site(entry_point, instance):
-    if (
-        entry_point
-        and entry_point.embedded_type == type_pb2.POI
-        and hasattr(entry_point.poi, 'properties')
-        and entry_point.poi.properties
-    ):
+def is_olymbic_poi(pt_object, instance):
+    if pt_object.embedded_type == type_pb2.POI:
+        if not (hasattr(pt_object.poi, 'properties') and pt_object.poi.properties):
+            return False
+    if pt_object.embedded_type == type_pb2.POI:
```

Comment:
I believe you can remove this line, you already did the test line 1262.

Reply:
:+1: